pkg/cmd/server/api/v1/types.go

```diff
@@ -15,6 +15,10 @@ type NodeConfig struct {
 	// If you're describing a set of static nodes to the master, this value must match one of the values in the list
 	NodeName string `json:"nodeName"`
 
+	// Node may have multiple IPs, specify the IP to use for pod traffic routing
+	// If not specified, network parse/lookup on the nodeName is performed and the first non-loopback address is used
+	NodeIP string `json:"nodeIP"`
```

Comment:
Can you make a more specific name?  Nodes also have to communicate how the master can contact them, so I don't want any confusion.